src/changelog.js

```diff
@@ -8,21 +8,28 @@ let spawnSync = require('child_process').spawnSync;
 let {filter, clone} = require('./utils')
 let Logger = require('./logger')
 
-let github = new GithubApi({
-  version: '3.0.0',
-  timeout: 10000,
-  protocol: 'https'
-});
+function authenticate({host, pathPrefix} = {}) {
+  const opts = {
+    timeout: 10000,
+    protocol: 'https'
+  };
+  if (host) {
+    opts.pathPrefix = pathPrefix || '/api/v3';
+    opts.host = host;
```

Comment:
[pathPrefix](https://www.npmjs.com/package/github#example) is required to access Enterprise github, however, it's reasonable to provide a default value.